_includes/header.html

```diff
@@ -17,25 +17,36 @@
         <div class="collapse navbar-collapse" id="navbar-collapse">
           <ul class="nav navbar-nav">
             <li class="{% if page.permalink == '/' %}active{% endif %}"><a href="{{ '/' | relative_url }}"><i class="fa fa-home"></i></span> Inicio</a></li>
+            <li class="{% if page.permalink == '/docentes/' %}active{% endif %}"><a href="{{ '/docentes/' | relative_url }}"><i class="fa fa-chalkboard-teacher"></i></span> Docentes</a></li>
             <li class="{% if page.permalink == '/regimen/' %}active{% endif %}"><a href="{{ '/regimen/' | relative_url }}"><i class="fa fa-tasks"></i></span> Régimen</a></li>
-            <li class="{% if page.permalink == '/bibliografia/' %}active{% endif %}"><a href="{{ '/bibliografia/' | relative_url }}"><i class="fa fa-book"></i></span> Bibliografía</a></li>
-            <li class="{% if page.permalink == '/material/' %}active{% endif %}"><a href="{{ '/material/' | relative_url }}"><i class="fa fa-paperclip"></i></span> Material</a></li>
-            <li class="{% if page.permalink == '/tps/' %}active{% endif %}"><a href="{{ '/tps/' | relative_url }}"><i class="fa fa-briefcase"></i></span> TPs</a></li>
             <li class="{% if page.permalink == '/calendario/' %}active{% endif %}"><a href="{{ '/calendario/' | relative_url }}"><i class="fa fa-calendar-alt"></i></span> Calendario</a></li>
-            <li class="{% if page.permalink == '/faq/' %}active{% endif %}"><a href="{{ '/faq/' | relative_url }}"><i class="far fa-question-circle"></i></span> FAQ</a></li>
-            <li class="{% if page.permalink == '/docentes/' %}active{% endif %}"><a href="{{ '/docentes/' | relative_url }}"><i class="fa fa-chalkboard-teacher"></i></span> Docentes</a></li>
-            <li class="{% if page.permalink == '/encuestas/' %}active{% endif %}"><a href="{{ '/encuestas/' | relative_url }}"><i class="fa fa-poll"></i></span> Encuestas</a></li>
-            <li class="{% if page.permalink == '/infraestructura/' %}active{% endif %}"><a href="{{ '/infraestructura/' | relative_url }}"><i class="fa fa-network-wired"></i></span> Infraestructura</a></li>
+            <li class="{% if page.permalink == '/tps/' %}active{% endif %}"><a href="{{ '/tps/' | relative_url }}"><i class="fa fa-briefcase"></i></span> TPs</a></li>
+            <li class="dropdown">
```

Comment:
Se puede agregar esto para que quede como active el menu de material, cuando se esta en bibliografia, faq, o material

```suggestion
            {% assign menu_material = "/bibliografia/, /material/, /faq/" | split: ", " %}
            <li class="dropdown {% if menu_material contains page.permalink  %}active{% endif %}">
```

Reply:
Le agregaría un par más (como la guía) pero me parece buena idea